frontend/packages/shared/src/hooks/mutations/useAddOptionListMutation.ts

```diff
@@ -14,11 +14,9 @@ export const useAddOptionListMutation = (org: string, app: string, meta?: Mutati
 
   return useMutation({
     mutationFn,
-    onSuccess: async () => {
-      await Promise.all([
-        queryClient.invalidateQueries({ queryKey: [QueryKey.OptionListIds, org, app] }),
-        queryClient.invalidateQueries({ queryKey: [QueryKey.OptionLists, org, app] }),
-      ]);
```

Comment:
Er det noen grunn til å fjerne `Promise.all` her?

Reply:
Jeg fjernet det fordi jeg antok at vi ikke trenger å avvente de før vi fortsetter kjøring

Reply:
Får du ikke noe varsel om at `Promise returned from invalidateQueries is ignored`? En måte å gå rundt det på, hvis vi virkelig ikke trenger å vente på eller bruke resultatet, er å sette `void` foran.

Reply:
I konsollen? Jeg ser hvertfall ingenting når jeg trigger mutasjonen lokalt 🤷 Men kan sikkert bare legge det tilbake igjen om du tror det er tryggest?

Reply:
I kodeverktøyet. Jeg får det i hvert fall i Intellij. Det er ikke noe farlig så lenge man vet hva man gjør, men jeg ville nok i hvert fall gjort noe for å tydeliggjøre at det er gjort slik med vilje. Jeg foreslår følgende:
```
      void queryClient.invalidateQueries({ queryKey: [QueryKey.OptionListIds, org, app] });
      void queryClient.invalidateQueries({ queryKey: [QueryKey.OptionLists, org, app] });
```